src/general/HelpScreen.cs

```diff
@@ -1,159 +0,0 @@
-﻿using System;
```

Comment:
Has all the content from the help screen been moved to the Thriveopedia? Just double checking as we definitely don't want to lose any.

Reply:
The `Mechanics` category is supposed to replace the help menu, and should even be more detailed in the end.

However, I didn't get around to creating most of the pages in that category yet, and the ones I _did_ create most likely have some style inconsistencies and missing links.

I would really appreciate if you looked over what I added in this category. https://wiki.revolutionarygamesstudio.com/wiki/Category:Mechanics

Reply:
I didn't check how the pages look in the game yet, but it would be extremely good to have a short summary equivalent to the current help menu with links to other wiki pages that then explain things more in-depth.

Reply:
That could work, the MechanicsRoot page functions as the stand-in for the help menu, and links to other pages. I could add a short general summary to the root page (though I don't know if it won't end up being too long).

Each mechanic page features a short description in the intro paragraphs, and then goes into each part of the mechanic in-depth in separate sections, or at least they should.

Reply:
I think there really needs to be a stage and each editor specific "help" page. Which would basically be most important information covered briefly. Otherwise the help button in the game doesn't have any sensible place to send the player to read stuff.

Reply:
In that case there could be a help category with each editor and stage, as well as linking to the mechanics category for specific mechanic explanations

Reply:
Pretty soon that'll require pages to be able to be in multiple places because it makes much more logical sense to put the stage specific help pages under their stage category so that it is next to the in-depth information and other information the player needs. When playing the game normally you won't want to read the industrial stage help page while on the microbe help page.

My suggestion for now is to put help pages under their stages as that is where they belong in terms of when the player is going to read them.

Reply:
What I could suggest, to avoid both extreme page nesting and providing too much information to the player in the current stage, is to have one big dropdown somewhere in the thriveopedia (in a place that is easily accessible) which will select certain stages and then display only the relevant pages. For example, when Microbe Stage is selected, only show the categories for Organelles, Mechanics, Editor Help and Stage Info. And similarly, when Industrial Stage is selected, only show pages for buildings, tech and hide the organelle category as that information is no longer relevant.

When opened, the thriveopedia would automatically select the current stage, links to pages outside the current stage would still open the page, but that page would not appear in the tree. 

I think this approach would minimise the tree clutter (as it is already becoming a problem in my experience). In this case the tree would look like:

- Wiki
  - Mechanics
    - Compounds, Patch Map, etc...
  - Stages
    - Microbe Stage
    - Microbe Stage Help 
  - Organelles
    - Mitochondria, Chloroplast, etc...

For the time being I can implement your suggestion.

Reply:
Ultimately I think the information should primarily be grouped by stages, this means that there needs to be like a mechanics category under each stage separately. A layout like that only really works so far as we only have one stage.

Other stages in the tree can start off collapsed to reduce the clutter. This is how a tree view usually works. It would be against tree view GUI conventions to have a separate control that might be hard to understand for hiding tree elements.

So to me the nesting should be like:

- Wiki
  - Microbe Stage
    - Quick Help (microbe)
    - Mechanics
      - Compounds
    - Organelles
      - Axon
  - Multicellular Stage
    - Quick Help (multicellular)  
    - Mechanics
       - Some stuff
    - Organs
    - Organelles (this would duplicate info in the microbe stage tree)

That way the in-game wiki would be organized like a real page tree that is organized around information the player needs at the same.

Reply:
Godot has a feature for tree elements that are also dropdowns. That would be the best compromise in my opinion. The dropdown would contain all the stages, and then would show only the pages in the respective stage, layed out in the same way you described. In this way, you wouldn't have duplicated pages, you'd only see what you need to see, all the while being integrated naturally into the tree

Reply:
I suppose that could work but there are also non-stage specific pages that would need to be always visible in the tree.